mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -389,7 +433,55 @@ func logModelGenerationSummary(modelToCompGenerateTracker *store.GenerticThreadS
 	utils.Log.Info(fmt.Sprintf("Generated %d models and %d components", totalAggregateModel, totalAggregateComponents))
 }
 
+func createJsonFile(resp *sheets.ValueRange) error {
+
+	var customResp []CustomValueRange
+
+	for _, row := range resp.Values[2:] {
+		if len(row) >= 15 && row[0] != "" {
+			customResp = append(customResp, CustomValueRange{
+				Model:                row[0].(string),
+				Version:              row[1].(string),
+				Kind:                 row[2].(string),
+				Type:                 row[3].(string),
+				SubType:              row[4].(string),
+				MetadataDescription:  row[5].(string),
+				Docs:                 row[6].(string),
+				MetadataStyles:       row[7].(string),
+				EvalPolicy:           row[8].(string),
+				SelectorsDenyFrom:    row[9].(string),
+				SelectorsDenyTo:      row[10].(string),
+				SelectorsAllowFrom:   row[11].(string),
+				SelectorsAllowTo:     row[12].(string),
+				CompleteDefinition:   row[13].(string),
+				VisualizationExample: row[14].(string),
+			})
+		}
+	}
+
+	jsonData, err := json.MarshalIndent(customResp, "", "    ")
+	if err != nil {
+		utils.Log.Error(err)
+		return nil
+	}
+
+	jsonFile, err := os.Create("../docs/_data/RelationshipsData.json")
```

Comment:
Where is the helpful error that is presented to the user to let know that this command assumes they are running it from within the root of a cloned Meshery repo?

Reply:
yes json is good to go